src/ck-cp/arrayRedistributor.h

```diff
@@ -387,10 +337,10 @@ class redistributor2D: public CBase_redistributor2D {
   void printArrays(){
 #if DEBUG > 2
     CkAssert(data_arrays.size()==2);
-    for(std::map<int,double*>::iterator diter = data_arrays.begin(); diter != data_arrays.end(); diter++){
+    for(auto diter = data_arrays.begin(); diter != data_arrays.end(); diter++){
```

Comment:
Why not just `for(auto &diter : data_arrays)`? (same below)

Reply:
Done